src/coreclr/tools/dotnet-pgo/PgoRootCommand.cs

```diff
@@ -0,0 +1,297 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Help;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Diagnostics.Tools.Pgo
+{
+    internal sealed class PgoRootCommand : RootCommand
+    {
+        public Option<string[]> InputFilesToMerge { get; } =
+            new(new[] { "--input", "-i" }, "Input .mibc files to be merged. Multiple input arguments are specified as --input file1.mibc --input file2.mibc") { IsRequired = true, Arity = ArgumentArity.OneOrMore };
```

Comment:
As I mentioned in the other comment, I think this and others should stay as `FileInfo`. E.g. I believe it can be:
```suggestion
        public Option<List<FileInfo>> InputFilesToMerge { get; } =
            new(new[] { "--input", "-i" }, "Input .mibc files to be merged. Multiple input arguments are specified as --input file1.mibc --input file2.mibc") { IsRequired = true, Arity = ArgumentArity.OneOrMore }.ExistingOnly();
```

Reply:
This is done for consistency with other RootCommands we have in this repos. The validation works well with primitive types.

Reply:
Not sure I understand. For example, `InnerDumpMain` is removing validation that the file exists, but where does that get validated after your change without the `FileInfo` and `ExistingOnly` that System.CommandLine supports?

Reply:
I'm ok with using strings instead of `FileInfo` assuming @davidwrighton is too, but using the built-in validation with `FileInfo` could be nice I think.

Reply:
> `InnerDumpMain` is removing validation that the file exists, but where does that get validated after your change

I considered such one-off validations redundant. Few lines after that validation, there is a call to `OpenMibcAsPEReader`, which uses `FileStream` and throws a standard file not found exception.

> `FileInfo` could be nice I think.

I agree. In ILC and crossgen2 rootcommands, we used strings last week. I have made this consistent with those. In a next pass, we can unify the style and use file/directory info in all (5-6) root commands in the repo. The idea is to keep these consistent as much as possible so when it's time to update the `System.CommandLine` package, we have a familiar style to work with.

Reply:
> I considered such one-off validations redundant.

Well, the person who wrote it apparently didn't and throwing an unhandled exception at the user is certainly not a good experience either. But this happens in a multitude of places so I'm fine with it.

Reply:
> throwing an unhandled exception at the user is certainly not a good experience either

We have a global exception handler in PgoRootCommand (line 258), so no exception is left unhanded during a subcommand execution.